### ISSUE TITLE
cfamily analyzer broken

### ISSUE THREAD

**Issue by jclsn** [asmodeus812/coc-sonarlint#3]
The current version btw did not work for me for C/C++. I had to update the analyzer with the latest version and then it worked. 

Would be great if you could do that as well. Here is the output when it doesn't work

```
Executing /home/jan/.config/coc/extensions/coc-sonarlint-data/jdk-17.0.8/javajre-linux-64/jre/bin/java -jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/server/sonarlint-ls.jar -stdio -analyzers /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonargo.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarjava.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarjs.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarphp.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarpython.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarhtml.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarxml.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonartext.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonariac.jar /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
[Info  - 21:00:36.986] SonarLint backend started, instance=org.sonarsource.sonarlint.core.rpc.impl.SonarLintRpcServerImpl@37d4349f
[Info  - 21:00:37.126] Skipping text document analysis because workspace folder is not synchronized yet "file:///home/jan/Workspace/C%2B%2B/constexpr/constexpr.cpp"
[Warn  - 21:00:37.127] Using ${workspaceFolder} variable in sonarlint.pathToCompileCommands is only supported for files in the workspace
[Info  - 21:00:38.605] Started embedded server on port 64120
[Error - 21:00:39.203] Unable to load plugin /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
[Error - 21:00:39.203] java.lang.IllegalStateException: Error while reading plugin manifest from jar: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:105)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.PluginInfo.create(PluginInfo.java:221)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginRequirementsChecker.checkRequirements(SonarPluginRequirementsChecker.java:67)
	at org.sonarsource.sonarlint.core.plugin.commons.PluginsLoader.load(PluginsLoader.java:75)
	at org.sonarsource.sonarlint.core.plugin.PluginsService.loadPlugins(PluginsService.java:145)
	at org.sonarsource.sonarlint.core.plugin.PluginsService.getEmbeddedPlugins(PluginsService.java:77)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.getOrCreateStandaloneEngine(AnalysisEngineCache.java:85)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.getOrCreateAnalysisEngine(AnalysisEngineCache.java:76)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.registerModuleIfLeafConfigScope(AnalysisEngineCache.java:149)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.sonarsource.sonarlint.core.analysis.AnalysisService.onConfigurationScopeAdded(AnalysisService.java:451)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:344)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:229)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:166)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:178)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:164)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:145)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:391)
	at org.sonarsource.sonarlint.core.ConfigurationService.didAddConfigurationScopes(ConfigurationService.java:65)
	at org.sonarsource.sonarlint.core.rpc.impl.ConfigurationRpcServiceDelegate.lambda$didAddConfigurationScopes$0(ConfigurationRpcServiceDelegate.java:36)
	at org.sonarsource.sonarlint.core.rpc.impl.AbstractRpcServiceDelegate.withLogger(AbstractRpcServiceDelegate.java:120)
	at org.sonarsource.sonarlint.core.rpc.impl.AbstractRpcServiceDelegate.lambda$notify$6(AbstractRpcServiceDelegate.java:112)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.file.NoSuchFileException: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:148)
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1396)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:717)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:251)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:180)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:346)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:317)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:283)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:97)
	... 32 more

[Error - 21:00:39.229] Unable to load plugin /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
[Error - 21:00:39.229] java.lang.IllegalStateException: Error while reading plugin manifest from jar: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:105)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.PluginInfo.create(PluginInfo.java:221)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginRequirementsChecker.checkRequirements(SonarPluginRequirementsChecker.java:67)
	at org.sonarsource.sonarlint.core.plugin.commons.PluginsLoader.load(PluginsLoader.java:75)
	at org.sonarsource.sonarlint.core.plugin.PluginsService.loadPlugins(PluginsService.java:145)
	at org.sonarsource.sonarlint.core.plugin.PluginsService.getEmbeddedPlugins(PluginsService.java:77)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.getOrCreateStandaloneEngine(AnalysisEngineCache.java:85)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.getOrCreateAnalysisEngine(AnalysisEngineCache.java:76)
	at org.sonarsource.sonarlint.core.analysis.AnalysisEngineCache.registerModuleIfLeafConfigScope(AnalysisEngineCache.java:149)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.sonarsource.sonarlint.core.analysis.AnalysisService.onConfigurationScopeAdded(AnalysisService.java:451)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:344)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:229)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:166)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:178)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:164)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:145)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:421)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:391)
	at org.sonarsource.sonarlint.core.ConfigurationService.didAddConfigurationScopes(ConfigurationService.java:65)
	at org.sonarsource.sonarlint.core.rpc.impl.ConfigurationRpcServiceDelegate.lambda$didAddConfigurationScopes$0(ConfigurationRpcServiceDelegate.java:36)
	at org.sonarsource.sonarlint.core.rpc.impl.AbstractRpcServiceDelegate.withLogger(AbstractRpcServiceDelegate.java:120)
	at org.sonarsource.sonarlint.core.rpc.impl.AbstractRpcServiceDelegate.lambda$notify$6(AbstractRpcServiceDelegate.java:112)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.file.NoSuchFileException: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:148)
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1396)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:717)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:251)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:180)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:346)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:317)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:283)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:97)
	... 32 more

[Info  - 21:00:39.576] Skipping text document analysis because workspace folder is not synchronized yet "file:///home/jan/Workspace/C%2B%2B/constexpr/constexpr.cpp"
[Error - 21:00:39.589] Unexpected error occurred in scheduled task
[Error - 21:00:39.589] java.util.concurrent.CompletionException: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Unhandled method sonarlint/listFilesInFolder
	at java.base/java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413)
	at java.base/java.util.concurrent.CompletableFuture.join(CompletableFuture.java:2118)
	at org.sonarsource.sonarlint.core.fs.ClientFileSystemService.initializeFileSystem(ClientFileSystemService.java:116)
	at org.sonarsource.sonarlint.core.commons.SmartCancelableLoadingCache.lambda$newValueAndScheduleComputation$2(SmartCancelableLoadingCache.java:99)
	at org.sonarsource.sonarlint.core.commons.DebounceComputer.lambda$scheduleComputationAsync$0(DebounceComputer.java:83)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.eclipse.lsp4j.jsonrpc.ResponseErrorException: Unhandled method sonarlint/listFilesInFolder
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleResponse(RemoteEndpoint.java:209)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:193)
	at org.sonarsource.sonarlint.core.rpc.protocol.SingleThreadedMessageConsumer.lambda$new$0(SingleThreadedMessageConsumer.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 3 common frames omitted

[Info  - 21:00:39.593] Analyzing file "file:///home/jan/Workspace/C%2B%2B/constexpr/constexpr.cpp"...
[Error - 21:00:39.612] Unable to load plugin /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
[Error - 21:00:39.612] java.lang.IllegalStateException: Error while reading plugin manifest from jar: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:105)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.PluginInfo.create(PluginInfo.java:221)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginRequirementsChecker.checkRequirements(SonarPluginRequirementsChecker.java:67)
	at org.sonarsource.sonarlint.core.plugin.commons.PluginsLoader.load(PluginsLoader.java:75)
	at org.sonarsource.sonarlint.core.client.legacy.analysis.SonarLintAnalysisEngine.restart(SonarLintAnalysisEngine.java:107)
	at org.sonarsource.sonarlint.core.client.legacy.analysis.SonarLintAnalysisEngine.<init>(SonarLintAnalysisEngine.java:90)
	at org.sonarsource.sonarlint.ls.EnginesFactory.lambda$createEngine$0(EnginesFactory.java:115)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:684)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:662)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2168)
	at org.sonarsource.sonarlint.ls.EnginesFactory.createEngine(EnginesFactory.java:114)
	at org.sonarsource.sonarlint.ls.standalone.StandaloneEngineManager.getOrCreateAnalysisEngine(StandaloneEngineManager.java:37)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeStandalone(AnalysisTaskExecutor.java:498)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.lambda$analyzeSingleModuleNonExcluded$20(AnalysisTaskExecutor.java:360)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeSingleModuleNonExcluded(AnalysisTaskExecutor.java:360)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeSingleModule(AnalysisTaskExecutor.java:314)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:232)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:197)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.lambda$analyze$2(AnalysisTaskExecutor.java:165)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:165)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.run(AnalysisTaskExecutor.java:137)
	at org.sonarsource.sonarlint.ls.AnalysisScheduler.lambda$analyzeAsync$1(AnalysisScheduler.java:261)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.file.NoSuchFileException: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarlintomnisharp.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:148)
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1396)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:717)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:251)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:180)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:346)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:317)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:283)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:97)
	... 28 more

[Error - 21:00:39.615] Unable to load plugin /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
[Error - 21:00:39.615] java.lang.IllegalStateException: Error while reading plugin manifest from jar: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:105)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.PluginInfo.create(PluginInfo.java:221)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginRequirementsChecker.checkRequirements(SonarPluginRequirementsChecker.java:67)
	at org.sonarsource.sonarlint.core.plugin.commons.PluginsLoader.load(PluginsLoader.java:75)
	at org.sonarsource.sonarlint.core.client.legacy.analysis.SonarLintAnalysisEngine.restart(SonarLintAnalysisEngine.java:107)
	at org.sonarsource.sonarlint.core.client.legacy.analysis.SonarLintAnalysisEngine.<init>(SonarLintAnalysisEngine.java:90)
	at org.sonarsource.sonarlint.ls.EnginesFactory.lambda$createEngine$0(EnginesFactory.java:115)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:684)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:662)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2168)
	at org.sonarsource.sonarlint.ls.EnginesFactory.createEngine(EnginesFactory.java:114)
	at org.sonarsource.sonarlint.ls.standalone.StandaloneEngineManager.getOrCreateAnalysisEngine(StandaloneEngineManager.java:37)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeStandalone(AnalysisTaskExecutor.java:498)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.lambda$analyzeSingleModuleNonExcluded$20(AnalysisTaskExecutor.java:360)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeSingleModuleNonExcluded(AnalysisTaskExecutor.java:360)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyzeSingleModule(AnalysisTaskExecutor.java:314)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:232)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:197)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.lambda$analyze$2(AnalysisTaskExecutor.java:165)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.analyze(AnalysisTaskExecutor.java:165)
	at org.sonarsource.sonarlint.ls.AnalysisTaskExecutor.run(AnalysisTaskExecutor.java:137)
	at org.sonarsource.sonarlint.ls.AnalysisScheduler.lambda$analyzeAsync$1(AnalysisScheduler.java:261)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.nio.file.NoSuchFileException: /home/jan/.config/coc/extensions/node_modules/coc-sonarlint/analyzers/sonarcfamily.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:148)
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1396)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:717)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:251)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:180)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:346)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:317)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:283)
	at org.sonarsource.sonarlint.core.plugin.commons.loading.SonarPluginManifest.fromJar(SonarPluginManifest.java:97)
	... 28 more

[Info  - 21:00:43.883] Found 0 issues
[Info  - 21:00:43.883] Analyzing file "file:///home/jan/Workspace/C%2B%2B/constexpr/constexpr.cpp"...
```

**Comment by asmodeus812** [comment]
Yeah, that could be the case, i have not seen it thus far, i am usually running the binaries off of Mason, that is why i am not even sure what the current sonarlint embeded version is, will update it.

**Comment by asmodeus812** [comment]
@jclsn can you give this one a try ?

**Comment by asmodeus812** [comment]
Published 0.0.11
